lib/rubocop/cop/mixin/active_record_helper.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    # A mixin to extend cops for ActiveRecord features
+    module ActiveRecordHelper
+      extend NodePattern::Macros
+
+      def_node_search :find_set_table_name, <<-PATTERN
```

Comment:
Can you use squiggly heredoc in this PR?

Reply:
Oops, I guess the cause is my old-style snippets. Thank you notice that!
https://github.com/pocke/dotfiles/blob/c5def33338c0a9f4b32086986c0fc4fd6250f4ff/snippets/ruby.snip#L73-L83